otter/util/zkpartitioner.py

```diff
@@ -67,9 +67,10 @@ def startService(self):
 
     def stopService(self):
         """Release the buckets."""
-        MultiService.stopService(self)
+        d = MultiService.stopService(self)
```

Comment:
Oh, nice catch!  Also, non-blocking, but I think the general suggestion is to avoid calling the superclass like this (since Python3 only has new style classes and this kind of superclass call may no longer work) and to use `super(Partitioner, self).stopService()`.  That only works if `Partitioner` inherits from both `MultiService` and `object`, though (which is Python3-compatible).